license-example-projects/public-license-examples/booz-allen-public-license-python/pyproject.toml

```diff
@@ -4,12 +4,13 @@ version = "1.3.0.dev"
 description = ""
 authors = ["Your Name <you@example.com>"]
 
-[tool.poetry.dependencies]
-python = "^3.11"
-
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 behave = "^1.2.6"
 black = ">=24.3.0"
+pylint = "^3.3.1"
```

Comment:
A: I think our general preference is ">=" for dev dependencies so we can pull in major updates -- wasn't sure if this was a particular exception.

Reply:
Yeah - there are a few of these, but I left them to focus on the point of the ticket.  The reason here is that when habushu adds these, it does it via `poetry add`, which uses the `^` by default.